tests/x11/evolution/evolution_prepare_servers.pm

```diff
@@ -30,9 +30,18 @@ sub run() {
         zypper_call("in dovecot", exitcode => [0, 102, 103]);
         zypper_call("rr dovecot_repo");
         save_screenshot;
-    }
-    else {
-        zypper_call("in dovecot", exitcode => [0, 102, 103]);
+    } else {
+        if (is_opensuse) {
+            # on both Leap and TW we need to enable repo-oss before in order to be able to install dovecot
+            my $distro = is_leap ? "distribution/leap/" . get_var('VERSION') : "tumbleweed";
+            zypper_call("ar -f http://download.opensuse.org/" . $distro . "/repo/oss repo-oss");
+            zypper_call("in dovecot", exitcode => [0, 102, 103]);
```

Comment:
This change disables calling ``zypper in dovecot`` for SLES, is that intentional?

Reply:
Good catch @baierjan :) no, that was not intended at all!